source/install/install-nginx.rst

```diff
@@ -0,0 +1,55 @@
+.. _install-nginx:
+
+Installing NGINX Server
+=======================
+
+In a production setting, use a proxy server for greater security and performance of Mattermost.
+
+The main benefits of using a proxy are as follows:
+
+  -  SSL termination
+  -  HTTP to HTTPS redirect
+  -  Port mapping ``:80`` to ``:8065``
+  -  Standard request logs
+
+**To install NGINX on Ubuntu Server:**
+
+1. Log into the server that will host the proxy and open a terminal window.
+
+2. Install NGINX.
+
+  ``sudo apt-get install nginx``
+
+3. After the installation is complete, verify that NGINX is running.
+
+  ``curl http://localhost``
+  
+  If NGINX is running, you see the following output:
+  
+  .. code-block:: html
+  
+    <!DOCTYPE html>
+    <html>
+    <head>
+    <title>Welcome to nginx!</title>
+    .
+    .
+    .
+    <p><em>Thank you for using nginx.</em></p>
+    </body>
+    </html>
+
+.. Note::
```

Comment:
This is additional, "nice to know" info that is not needed to complete the task. It can be in a note, or it can be left out altogether.